app/apps/workflow/views.py

```diff
@@ -355,6 +358,39 @@ def district_names(self, request):
         serializer = serializers.ListSerializer(queryset, child=serializers.CharField())
         return Response(serializer.data)
 
+    @extend_schema(
+        description="Gets the SummonTypes associated with the given  task id",
+        responses={status.HTTP_200_OK: SummonTypeSerializer(many=True)},
+    )
+    @action(
+        detail=True,
+        url_path="summon-types",
+        methods=["get"],
+    )
+    def summon_types(self, request, pk):
+        paginator = LimitOffsetPagination()
+        caseUserTask = self.get_object()
+        current_workflow_version = caseUserTask.workflow.workflow_version
+        theme = caseUserTask.case.theme
+
+        # Summon types with workflow option "besluit" are only available from version "6.3.0"
+        lts_workflow_version = "6.3.0"
+        lts_workflow_option = "besluit"
+
+        if version.parse(current_workflow_version) >= version.parse(
+            lts_workflow_version
+        ):
```

Comment:
Ik zou verwachten dat lts_workflow_version ergens anders als constante gedefineerd staat, en dat de logica om te bapeln of de huidige workflow version groter is, ergens in een ander class of model of generieke helper functie staat.